### PR TITLE
Fix: policy-violation-desc.html 

### LINKED ISSUE #885: Rename Creation Date to Date Created, add time stamp
On the Policy Violations Details page, rename Creation Date to Date Created, and add the timestamp of when it was created.

### PR DESCRIPTION
**Pull request by Preston-Dailey**
# Description

Changed Creation date to Date Created and added a time stamp of the current day. 

Fixes #885

## Fix

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Viewed the text in a view window it was a text only change.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My commit message/PR follows the contribution guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# **Other information**:

This is my first ever pull request.  I assumed time stamp meant current time.  Thanks for letting me try to contribute, hopefully I can learn from this.

### CODE REVIEW

**Comment by Preston-Dailey** [comment]
Thanks for taking the time to check this.  Let me know anything I can do to make it better or contribute in another area.

File: webapp/src/app/pacman-features/secondary-components/policy-violation-desc/policy-violation-desc.component.html [hunk 1]
```diff
@@ -68,7 +68,7 @@
     </div>
     <div class="policy-violation-label flex flex-col" *ngIf="violationData.violationCreatedDate">
         <div class="policy-violation-label-header capitalize">
-            Creation date
+            Date Created 2/28/2023
```
Comment (plyubich) [inline webapp/src/app/pacman-features/secondary-components/policy-violation-desc/policy-violation-desc.component.html:71]: The column name should not contain a hardcoded date value. Please remove 2/28/2023.

The actual value for this is coming from line 74

**Review by plyubich — CHANGES_REQUESTED** [review]

**Comment by Preston-Dailey** [comment]
Awesome, thanks John!

**Review by plyubich — APPROVED** [review]
looks good

**Pull request merged**